code/pilotfile/plr_convert.cpp

```diff
@@ -126,16 +126,16 @@ plr_data::~plr_data()
 void pilotfile_convert::plr_import_controls()
 {
 	int idx;
-	config_item con;
+	CCI con;
 
 	unsigned char num_controls = cfread_ubyte(cfp);
 
 	if ( !num_controls ) {
 		return;
 	}
 
-	// it may be less than 118, but it shouldn't be more than 118
-	if (num_controls > 118) {
+	// TODO: Currently only checks for hardcoded control bindings. Scripted controls will be sliced out in the pilot file
+	if (num_controls > CCFG_MAX) {
```

Comment:
Can't do that. The magic number there is set to match the number of controls that were used in the old plr file format. Changing it will corrupt the data on conversion.

I'm pretty sure that I put a warning in there somewhere about not changing the magic numbers.

Reply:
Reverted.  Also added a comment to say "can't touch this"